src/Apps/Artist/Routes/AuctionResults/ArtistAuctionResultItem.tsx

```diff
@@ -28,13 +28,8 @@ export class ArtistAuctionResultItem extends Component<Props> {
   render() {
     const { lastChild } = this.props
     return (
-      <Subscribe to={[AppState]}>
-        {({ state }) => {
-          const {
-            mediator,
-            system: { currentUser },
-          } = state
-
+      <ContextConsumer>
+        {({ currentUser, mediator }) => {
```

Comment:
@ds300 - I removed `WithMediator` in this PR since `<ContextConsumer>` will be the new form, and things passed into `context` upon init will be accessible here